subsys/mpsl/cx/CMakeLists.txt

```diff
@@ -8,5 +8,5 @@ zephyr_library()
 
 zephyr_library_sources_ifdef(CONFIG_MPSL_CX mpsl_cx.c)
 zephyr_library_sources_ifdef(CONFIG_MPSL_CX_THREAD thread/mpsl_cx_thread.c)
-zephyr_library_sources_ifdef(CONFIG_MPSL_CX_BT_3WIRE bluetooth/mpsl_cx_bluetooth.c)
+zephyr_library_sources_ifdef(CONFIG_MPSL_CX_BT_3WIRE bluetooth/mpsl_cx_3w_bluetooth.c)
```

Comment:
this should be a fixup of another commit or belong in a "Rename 3-wire coex" or something commit where you put all the kconfig/filename renaming